docs/pages/access-controls/guides/access-requests-slack.mdx

```diff
@@ -0,0 +1,347 @@
+---
+title: Teleport integration with Slack
+description: This guide explains how to set up a Slack plugin for Teleport for privilege elevation approvals.
+h1: Teleport Slack Plugin Setup
+---
+
+This guide examines how to configure Teleport for use with Slack.
+
+Slack integration allows you to treat Teleport access and permission requests within Slack messages as well as through the command line.
+
+<Admonition type="Note" title="Access Requests">
+  Access Requests are now supported in both the Teleport Open Source and Enterprise Editions.
+</Admonition>
+
+#### Example Slack request
+
+<video controls>
+  <source src="../../../img/enterprise/plugins/slack/slack.mp4" type="video/mp4" />
+  <source src="../../../img/enterprise/plugins/slack/slack.webm" type="video/webm" />
+  Your browser does not support the video tag.
+</video>
+
+## Prerequisites
+
+This guide assumes that you have:
+
+- Teleport (=teleport.version=).
+- Admin privileges with access to [`tctl`](https://gravitational.com/teleport/docs/cli-docs/#tctl).
+- Slack Admin Privileges to create an app and install it to your workspace.
+
+<Admonition type="Tip" title="tip">
+  Read [Getting Started](../../getting-started.mdx) to install Teleport and configure it with a user and role.
+</Admonition>
+
+## Step 1/4. Create a user and role for access
+
+1. Log into your Teleport Authentication Node. The Authentication Node is where you'll most likely perform administrative tasks and run the powerful `tctl` ("tea-kettle") admin tool. 
+
+2. Create a new user and role with only API access to the `access_request` API. 
+  
+   The following YAML file specifies a non-interactive bot `access-plugin` user and role.
+
+   ```yaml
+   kind: user
+   metadata:
+     name: access-plugin
+   spec:
+     roles: ['access-plugin']
+   version: v2
+   ---
+   kind: role
+   version: v3
+   metadata:
+     name: access-plugin
+   spec:
+     allow:
+       rules:
+         - resources: ['access_request']
+           verbs: ['list', 'read']
+         - resources: ['access_plugin_data']
+           verbs: ['update']
+       # Teleport currently refuses to issue certs for a user with 0 logins.
```

Comment:
I would check, but this should work without logins already

Reply:
Confirmed it does. Thanks!